idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -0,0 +1,1768 @@
+                return blk.yi_MEA[t, x] == 0
+            else:
+                return blk.enhancement_factor[t, x] * (1 - blk.yb_CO2[t, x]) == \
+                    blk.Hatta[t, x] * blk.yi_MEA[t, x] * \
+                    (1 - blk.yeq_CO2[t, x])
+
+        @self.Constraint(self.t,
+                         self.liquid_phase.length_domain,
+                         doc='Enhancement factor lower bound ')
+        def E3_eqn(blk, t, x):
+            if x == self.liquid_phase.length_domain.last():
+                return Constraint.Skip
+            else:
+                return 1 - blk.enhancement_factor[t, x] <= 0.0
+
+        # ---------------------------------------------------------------------
+        # Boundary Conditions
+        # Enthalpy balance
+
+        @self.Constraint(self.t,
+                         doc='Boundary condition for vapor temperature @ inlet ')
+        def vap_BC_temperature(blk, t):
+            return blk.vap_in_temperature[t] ==\
+                blk.vapor_phase.properties[t, 0].temperature
+
+        @self.Constraint(self.t,
+                         doc='Boundary condition for liquid temperature @ inlet ')
+        def liq_BC_temperature(blk, t):
+            return blk.liq_in_temperature[t] ==\
+                blk.liquid_phase.properties[t, 1].temperature
+
+        # Mass balance-mole_frac
+        @self.Constraint(self.t, liq_comp,
+                         doc='Boundary condition for liquid mole fraction @ inlet ')
+        def liq_BC_mole_frac(blk, t, j):
+            return blk.liq_in_mole_frac[t, j] ==\
+                blk.liquid_phase.properties[t, 1].mole_frac_comp[j]
+
+        @self.Constraint(self.t, vap_comp,
+                         doc='Boundary condition for vapor mole fraction @ inlet ')
+        def vap_BC_mole_frac(blk, t, j):
+            return blk.vap_in_mole_frac[t, j] ==\
+                blk.vapor_phase.properties[t, 0].mole_frac_comp[j]
+
+        # Mass balance-flow-mol
+        @self.Constraint(self.t,
+                         doc='Boundary condition for vapor flow @ inlet ')
+        def vap_BC_flow_mol(blk, t):
+            return blk.vap_in_flow[t] ==\
+                blk.vapor_phase.properties[t, 0].flow_mol
+
+        @self.Constraint(self.t,
+                         doc='Boundary condition for liquid flow @ inlet ')
+        def liq_BC_flow_mol(blk, t):
+            return blk.liq_in_flow[t] ==\
+                blk.liquid_phase.properties[t, 1].flow_mol
+
+        # Pressure balance
+        @self.Constraint(self.t,
+                         doc='Boundary condition for  pressure @ inlet ')
+        def vap_BC_pressure(blk, t):
+            return blk.pressure_bottom[t] ==\
+                blk.vapor_phase.properties[t, 0].pressure
+
+    # ==========================================================================
+    # Model initialization routine
+    def initialize(blk, method='load_json', *args, **kwargs):
```

Comment:
The main `initialize` method should be the `_default_init` method, with the load from json being a separate alternative. Also, seeing as we have to and from json utilities in `idaes.core.util`, I wonder if we really need to duplicate that here.

Reply:
I don't under what you mean by duplicating to and from JSON utilities. Initialization of the packed column is not trivial and these utilities have been used to help the user have a saved solution so that this process is only repeated when other alternatives have failed.  While ``` _default_init ``` method is the main initialization, in the documentation , users will be advised to use the homotopy function for input parameters far away from the normal conditions and if it fails then, they can then use the default initialization method. This will be covered under a section for tips on initialization.  Moreover, the user still has the option of calling the default method directly if they want.

Reply:
IDAES already has utility functions for saving and loading json representations of models: https://github.com/IDAES/idaes-pse/blob/main/idaes/core/util/model_serializer.py.

Thus, there should be no need ofr custom functions in this model.

Secondly, the default initialization method should be the one named `initialize`. I.e. `_default_init` should be the `initialize` method, and the save/load json should have the different name.

Reply:
Sorry, I see what you are doing now - I think you need to comment this better to make it clear, as the call to `_default_init`  got lost in the code.

However, I would still suggest removing the to and from JSON parts here, as users can call those themselves from the flowsheet (and without you needing all the code for finding paths). Also, I don not think the initialization should save a JSON by default - that should be something the user asks for, especially because they should chose where to save the file (rather than a default location).